src/openassetio-core/include/openassetio/hostApi/ManagerFactory.hpp

```diff
@@ -28,6 +30,54 @@ OPENASSETIO_DECLARE_PTR(ManagerFactory)
 class OPENASSETIO_CORE_EXPORT ManagerFactory final {
  public:
   using Identifiers = std::vector<Str>;
+
+  /**
+   * Simple struct containing the default configuration details of a
+   * potential @ref manager implementation.
+   */
+  struct ManagerDetail {
```

Comment:
[question] We're not fussed about the MSVC struct issues here are we - as we only use it by value?

Reply:
The `struct` vs. `class` issue only surfaces if we forward declare it, which I'm not sure is even possible for child classes. Do you think we should make it `class` regardless, for consistency?